### DISCUSSION TITLE
How can I get my middleware to access the params within my URL after the #?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [nuxt/nuxt#10567] [Questions (Nuxt 2)] [answered]
A third-party API redirects me to something like http://localhost:3000/#access_token= and I want to access the access_token within my middleware, but am unsure how.

**Comment by danielroe** [comment, marked as answer]
It's not possible to do this within a server side environment, FYI, as the browser doesn't send anything after the hash to the server.

**Reply by CodyBontecou** [reply]
Ah okay, good to know! Thank you. I guess I can use `useRoute` for this.
